x-pack/docs/en/security/securing-communications/tls-http.asciidoc

```diff
@@ -52,29 +67,29 @@ xpack.security.http.ssl.certificate_authorities: [ "/home/es/config/ca.crt" ] <3
     must be a location within the {es} configuration directory.
 --
 
-. If you secured the node's certificate with a password, add the password to
-your {es} keystore:
-
-** If the signed certificate is in PKCS#12 format, use the following commands:
+.. If you secured the keystore with a password, add that password to a secure
+configuration setting in {es}:
```

Comment:
```suggestion
setting in {es}:
```

I think we call them `secure settings` ( as opposed to `secure configuration settings`)  elsewhere